### DISCUSSION TITLE
Is it possible to create new note with Templater without popup windows asking "Which template to use?"?

### DISCUSSION THREAD

**Discussion by MikeKholomeev** [SilentVoid13/Templater#1348] [Help] [answered]
Hi,
So I want to create a button, with a plugin Commander, that should create a new note from a specific template.

I don't know how to create a command for Templater "Use this template to create new note"

Is it possible?

**Comment by Zachatoo** [comment, marked as answer]
Yes, there's a hotkeys section in Templater settings which lets you create a command for a template.

**Reply by MikeKholomeev** [reply]
Thanks.
Yes, it did the trick. But i had to rewrite the template code and get rid of AutoMover plugin, coz a soup of 3 plugins made my head spin.
